kore/src/Kore/Internal/Pattern.hs

```diff
@@ -218,9 +225,78 @@ fromTermLike term
   | otherwise =
     Conditional
         { term
-        , predicate = Predicate.makeTruePredicate
+        , predicate = Predicate.makeTruePredicate (termLikeSort term)
         , substitution = mempty
         }
 
+fromTermLikeUnsorted
```

Comment:
`Unsorted` and `_` at the end of a function name mean the same thing, right?

Reply:
Yes, though I find the "Unsorted" thing to be somewhat more readable. I used "_" for the predicate stuff since it had a clear relation to the mk* term constructors, but I wasn't sure that here it's similarly clear. If you think I should use underscores, I will.

Reply:
I understand, makes sense in this context. I was just reviewing the code and was confused why the convention changed, but as you say, we don't have such a convention for `Pattern`.

Reply:
It turned out that, after fixing all the broken tests, this was used only once in the entire project, so I removed it.